app/schemas/user.py

```diff
@@ -1,8 +1,40 @@
 from typing import Optional
-
 from pydantic import BaseModel, EmailStr
+from datetime import date
+from app.models.user import User as DBUser
 
 
 class UserLogin(BaseModel):
     username: EmailStr
     password: str
+
+
+class UserCreate(BaseModel):
+    first_name: str
+    last_name: str
+    email: EmailStr
+    password: str
+    user_type: str
+    date_of_birth: date
+
+
+class UserUpdate(BaseModel):
+    first_name: Optional[str] = None
+    last_name: Optional[str] = None
+    email: Optional[EmailStr] = None
+    password: Optional[str] = None
```

Comment:
We dont want to allow users to update password. This would be separeta functionality called change_password